jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -40,6 +43,15 @@ class JWTTools(
 ) {
     private val notEmpty: (String) -> Boolean = { !it.isEmpty() }
 
+    init {
+        // if no resolvers are registered use default resolvers
+        if (!UniversalDID.hasResolver()) {
```

Comment:
I think this should be a little more flexible instead of the all or nothing approach.
If there is no resolver for a known DID method, register the default one, otherwise, leave it alone.